app/src/main/res/layout/activity_main.xml

```diff
@@ -60,6 +60,17 @@
                 android:layout_height="44dp"
                 android:hint="@string/add_list_hint"
                 />
+
+            <com.joanzapata.iconify.widget.IconTextView
+                android:id="@+id/voice_entry_icon"
+                android:text="@string/voice_icon"
+                android:layout_gravity="end|center_vertical"
+                style="@style/list_item_button"
```

Comment:
I don't think we should use this style. We're not within a list and we don't need the centerVertical attribute within it (that's a RelativeLayout only attribute). Let's just inline those attributes.

Reply:
![image](https://user-images.githubusercontent.com/25705658/69834124-2324a600-1241-11ea-97d3-85ae25d7a2f7.png)

![image](https://user-images.githubusercontent.com/25705658/69834129-2c157780-1241-11ea-8420-7d2637237cf3.png)
this is the difference between the center and without the center

Reply:
Yeah, we still need the layout_gravity of center_vertical|end (for FrameLayouts), but we don't need the one within the style (for RelativeLayouts).

Reply:
Got it, I saw it but I think it will not make a difference so I kept it, but putting them inline it will be better and it will reduce the unused code.